sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/data/RemoteGrpcPortWrite.java

```diff
@@ -60,6 +60,10 @@ public static RemoteGrpcPortWrite fromPTransform(PTransform pTransform)
     return writeToPort(inputPCollectionId, port);
   }
 
+  public static boolean isSinkPtransform(PTransform pTransform) {
+    return URN.equals(pTransform.getSpec().getUrn());
```

Comment:
URN is already publicly exposed, reference it directly from the current callers of isSinkPTransform.

Reply:
Done